src/sagemaker_training/mpi.py

```diff
@@ -305,23 +305,22 @@ def run(self, wait=True, capture_error=False):
         logging_config.log_script_invocation(cmd, self._env_vars)
 
         training_env = environment.Environment()
-        exception_classes = get_modelparallel_exception_classes()
+        exception_classes = []
+        exception_classes += process.get_debugger_exception_classes()
+        if training_env.is_modelparallel_enabled:
+            exception_classes += get_modelparallel_exception_classes()
```

Comment:
If there are no exception classes found in sm modelparallel and debugger, it would result in `[errors.ExecuteUserScriptError, errors.ExecuteUserScriptError]`. Could you please take a look and fix this?